include/evmc/evmc.hpp

```diff
@@ -27,6 +31,22 @@ class result : private evmc_result
     using evmc_result::output_size;
     using evmc_result::status_code;
 
+    /// Creates the result from the provided arguments.
+    ///
+    /// The provided output is copied to memory allocated with malloc()
+    /// and the evmc_result::release function is set to one invoking free().
+    ///
+    /// @param _status_code  The status code.
+    /// @param _gas_left     The amount of gas left.
+    /// @param _output_data  The pointer to the output.
+    /// @param _output_size  The output size.
+    result(evmc_status_code _status_code,
```

Comment:
In other functions you don't use underscore prefix for parameter names

Reply:
We have to somehow avoid field names shadowing here.